app/api/upload/routes.js

```diff
@@ -39,6 +41,7 @@ const storeFile = (file, cb) => {
   });
 };
 
+/*eslint-disable max-statements*/
```

Comment:
@RafaPolit i think we should disable only when it makes sense and it is justified, max-statements is there for a reason, if it is because the linter adds to much noise to the code we should remove it before commit, but we should definitely refactor this.

Reply:
The reason that disable is there, is because the function that receives app and defines each route, has more than 10 functions inside, the error/warning appeared when I added a route, I think it's still possible to move some const definitions outside of that function so they don't reach 10, but the problem will appear again if we define more than 10 routes in a single file.